### DISCUSSION TITLE
Welcome to DevsHelpDevs, if you want to help, please introduce yourself

### DISCUSSION THREAD

**Discussion by escamoteur** [DevsHelpDevs/DevsHelpDevs#1] [General]
Hi,
I'm Thomas. I'm a freelance Flutter developer with over 30 years of IT experience in different fields.
Unfortunately I have no web development at all so I probably can't help with coding but will help to manage this project.

If you want to join this effort it would be great if you could introduce yourself and tell us how you think you can contribute to the project.

**Comment by sameeerkashyap** [comment]
Hi, I'm Sameer from Bangalore, India.
I'm a final year undergrad doing Bachelors in Computer Science Engineering. 
I'm very active amongst the student and flutter community with a tremendous love for flutter as it has given me everything I've wanted.
I'm a Google Developer Student Clubs Lead and an MLH Fellow for Spring 21' 
Trying to learn, collaborate and build great things 🛠

**Reply by biggaji** [reply]
Hi, I’m Tobi Ajibade from Ibadan, Nigeria.
Am a software engineer!
I specialize in building web applications using web technologies.
I hope to contribute to the development of these web application and other areas in which i can help too.
Thanks!

**Reply by Jhanbury** [reply]
Hi, I'm John from Ireland.
I'm a software engineer, I primarily build mobile and web apps. 
I work in .NET stack and Azure but have some experience with other stacks like flutter and Firebase.
More than happy to adapt to a new stack that suits the team better.
Would love to help out in whatever way I can 👍

**Reply by escamoteur** [reply]
Awesome. Please read the platform description https://github.com/DevsHelpDevs/DevsHelpDevs/blob/master/user-specification.md and share your thoughts here in the discussions.
Also please tell us which technologies you are comfortable with.

**Reply by julian-King22** [reply]
Kingsolomon Ifeanyi. Nigerian. Specialize in building web applications (JavaScript, Ruby on Rails)
I love to write code
I don't have any other life😄😄
In areas in which i can help, I would love to commit

**Reply by escamoteur** [reply]
Hi, 
awesome. I heard also a lot of good stuff about ruby.
Lets see what will use in the end.

**Comment by dev-darrell** [comment]
Hi, I'm Adnah from Nigeria.
I'm a certified Android developer proficient in building native apps with Java & Kotlin.

I'll love to help out wherever I can 😃

**Reply by StephenLadoye** [reply]
Okay, thanks

**Comment by shyndman** [comment]
Hi there,

I'm Scott, from Canada 🇨🇦 . I've done a little bit of everything, and a bunch of web dev (frontend, and backend in a bunch of languages/frameworks). I'm really into Flutter right now.

I have quite a bit of spare time at the moment, and I think this sounds interesting. Let's do it!

**Reply by StephenLadoye** [reply]
Okay, thanks

**Comment by mugisha23** [comment]
Hi,
I'm Michael From Burundi.
I'm a freshman undergrad doing IT.
I'm currently learning frontend developement with react 've also build some small projects.

I'm really interested in contributing to this with my knowledge.

**Reply by StephenLadoye** [reply]
Okay! Nice. Lets connect on Twitter @StephenLadoye If you need one year free access on pluralsight.

**Reply by clefayomide** [reply]
> Okay! Nice. Lets connect on Twitter @StephenLadoye If you need one year free access on pluralsight. 

Hello sir. I'm Ayomide, a final year undergraduate student interested in tech. I was also a beneficiary of the devshelpdevs project. I'm currently interning at theZuriTeam. Please I'm very much interested in the one year pluralsight access if it's still available. Thanks

**Comment by ShadKip** [comment]
My name is Kibet from Kenya. I am a junior Dev with capabilities in web development using React Js, Python Django Framework. Twitter handle @kibet_too. Supporting the cause.

**Reply by escamoteur** [reply]
great to meet you

**Reply by StephenLadoye** [reply]
@ShadKip You are welcome on board

**Comment by ayoolafelix** [comment]
Hi, I'm Felix, I'm a designer of all sorts, how can I help?

**Reply by escamoteur** [reply]
Hi Felix,

it would be awesome to get a nice project logo.
Also we have to come up with a general design for the system. please read:
https://github.com/DevsHelpDevs/DevsHelpDevs
https://github.com/DevsHelpDevs/DevsHelpDevs/blob/master/user-specification.md

**Comment by artemesian** [comment]
Hi I'm Angelo from Cameroon, 
Web and Mobile Developer & UI Designer(AdobeXD)

Would be glad to contribute to this great project

Thanks

**Comment by escamoteur** [comment]
Hi,

I created a Slack channel for this project to make communication a bit easier
https://join.slack.com/t/devshelpdevs/shared_invite/zt-mcsbfd9s-J4xhhmuMFouAzTf2GTCffw

**Comment by StephenLadoye** [comment]
Hi everyone,

Stephen Ladoye is here. Nice to connect and meet everyone here.

I'm based in Lagos, Nigeria. I'm a self taught software and IoT Engineer.

I'm a founder of Stephen Ladoye Foundation.

I will be handling Media and communications for DevsHelpDevs

In the mean time, if you are interested in one year free access on Pluralsight. Kindly hit me up. 

Cheers

**Comment by escamoteur** [comment]
I would like to make a zoom call with all of you tomorrow or Friday some time between 3pm and 7pm.
Which time works best for you. Please reply on Slack

**Comment by escamoteur** [comment]
Hi friends, I invite all of you to our first zoom meeting on Friday at 3pm (CET)  please try to join, so that we all get to know each other better.
https://us02web.zoom.us/j/82131008942?pwd=UUtnc0gyeDNVbTdKU2ljSktOWDByZz09

**Comment by StephenLadoye** [comment]
Hello,

Kindly check the email and resend.

Please be specific about the timing and date.


Yours,

Stephen

**Reply by shyndman** [reply]
Hey @StephenLadoye, the Zoom meeting is at 3pm on Friday (CET). 

If your profile is up to date, I think you're in the same timezone, so no math necessary.

**Comment by 0xdanny** [comment]
Hello. I'm Danny from Nigeria, a mobile application developer and security engineer.

Would love to help in whatever way possible!

**Comment by StephenLadoye** [comment]
Thank you for your response.

**Comment by seunbayo** [comment]
Am I late for the party?

I am seun by name, a Javascript developer (emberJs) looking forward to the transition to Vue.
I love and do more projects management this days and I am excited about this community

**Comment by Evavic44** [comment]
Hello everyone, I'm Victor Eke, a Frontend Web Developer proficient in HTML and CSS. Currently learning and contributing to open source projects. Pleased to meet you all. 🆒 👍🏾

**Comment by anassboutaline** [comment]
Hello everyone!

My name is Anass, i'am a 10 years experienced web developer, i'am really intrested in contributing,
Technologies: Php or nodejs for backend, React or vanilla JS for front end, and mySQL, sqlite, or postgresql.
Thank you

**Reply by biggaji** [reply]
Welcome onboard @mjprogramation

**Reply by anassboutaline** [reply]
Thank you @biggaji
